backend/src/book/model.spec.js

```
@@ -0,0 +1,89 @@
+import { expect } from 'chai';
+import db from '../sequelize/configureSequelize.js';
+import { ValidationError } from 'sequelize';
+import _ from 'lodash';
+
+const rawUser = {
+  email: 'john@smith.com',
+  firstName: 'John',
+  lastName: 'Smith',
+  password: 'v3rySecret',
+};
+
+const rawBook = {
+  title: 'Javascript for Humans',
+  copies: 1,
+  author: 'John Smith',
+  ISBN: '978-0345-410085',
+  meta: {
+    language: 'polish',
+  },
+  coverUrl: 'http://www.internet.com/cover.GIF',
+};
+
+const wrongBook = {
+  title: 'Javascript for Humans',
+  copies: 'one',
+  author: 'John Smith',
+  ISBN: '978-0345-419085',
+  meta: 'data',
+  coverUrl: 'ftp://www.internet.com/cover.sh',
+};
+
+describe('Book model', () => {
```

`Book model belongs to User should throw a ValidationError when User is not provided`
`Book model ISBN should remove all the -`
`Book model ISBN should prefix ISBN10 to get a ISBN13`
`Book model validation should save when data is valid`
`Book model validation should set shelfId to 1 before validating`
`Book model validation should throw an error when data is invalid`

which dosn't make much sense

Maby better something like this:
```
DESCRIBE Book model
  IT should remove all the `-` signs from ISBN number
  IT should prefix ISBN10 to get a ISBN13
  IT should set shelfId to 1 before validating`
  IT should throw a ValidationError when User is not provided`
  IT should throw ValidationError when book data is invalid`
  IT should save a book when data is valid
```
In BDD we should test bahavior of the object / function, not specific internal methods, so one DESCRIBE in this case is enough.
Also, we should be able to read test descriptions as sentences.